django/bossspatialdb/views.py

```diff
@@ -79,13 +79,22 @@ def get(self, request, collection, experiment, channel, resolution, x_range, y_r
         else:
             iso = False
 
+        #Default access_mode to cache on the server side in case it is not explicitly defined in an API call. 
+        access_mode = "cache"
+        #Translation from no-cache boolean param to access_mode param for backwards compatability. 
         if "no-cache" in request.query_params:
             if request.query_params["no-cache"].lower() == "true":
-                no_cache = True
+                access_mode = "no_cache"
+            elif request.query_params["no-cache"].lower() == "false":
+                access_mode = "cache"
```

Comment:
What if `no-cache` is other than `true` or `false`? Should it be a `400 Bad Request` response?

Reply:
The default access mode for get_cutout should be no-cache.  If you have both no-cache and an access-mode in the URL if they don't match, it should be a 400.

Reply:
@sandyhider I thought we had said that we only wanted to default to no_cache on the client side. On the server side. We default to using the cache. Has this changed?

Reply:
@sandyhider  If both no-cache and access_mode are specified in the url I think it would be better to just raise a 400 error with a message asking to use the access_mode param since no-cache will be deprecated.

Reply:
@derek-pryor Done